workflow/envs/pysam.yaml

```
@@ -1,4 +1,4 @@
 channels:
   - bioconda
 dependencies:
-  - pysam ==0.15.4
+  - pysam <=0.16
```

Here, I would probably instead pin both pysam and python to the most current available working version, so:
```suggestion
  - pysam ==0.16
  - python ==3.7
```